Firestore/core/src/firebase/firestore/core/user_data.mm

```diff
@@ -175,9 +176,8 @@
   absl::string_view designator{RESERVED_FIELD_DESIGNATOR};
   if (write() && absl::StartsWith(segment, designator) &&
       absl::EndsWith(segment, designator)) {
-    FSTThrowInvalidArgument(@"Document fields cannot begin and end with %s%s",
-                            RESERVED_FIELD_DESIGNATOR,
-                            FieldDescription().c_str());
+    ThrowInvalidArgument("Document fields cannot begin and end with %s%s",
+                         RESERVED_FIELD_DESIGNATOR, FieldDescription().c_str());
```

Comment:
Optional nit: I don't think the call to `c_str` is necessary anymore?

Reply:
Missed one, doh!